src/shared/lib/comparison/AnalysisStore.ts

```diff
@@ -0,0 +1,383 @@
+import { remoteData } from 'cbioportal-frontend-commons';
+import {
+    CancerStudy,
+    Gene,
+    Mutation,
+    MutationCountByPosition,
+    Sample,
+} from 'cbioportal-ts-api-client';
+import { computed, makeObservable, observable } from 'mobx';
+import _ from 'lodash';
+import { compileMutations } from '../../../pages/resultsView/ResultsViewPageStoreUtils';
+import internalClient from '../../api/cbioportalInternalClientInstance';
+import { AppStore } from '../../../AppStore';
+import {
+    evaluateMutationPutativeDriverInfo,
+    fetchOncoKbCancerGenes,
+    fetchOncoKbDataForOncoprint,
+    filterAndAnnotateMutations,
+    getGenomeNexusUrl,
+    makeGetOncoKbMutationAnnotationForOncoprint,
+    makeIsHotspotForOncoprint,
+    ONCOKB_DEFAULT,
+} from 'shared/lib/StoreUtils';
+import MobxPromise, { MobxPromiseUnionTypeWithDefault } from 'mobxpromise';
+import { AnnotatedMutation } from '../../../pages/resultsView/ResultsViewPageStore';
```

Comment:
We should move `AnnotatedMutation` type somewhere else. It is not ideal to have a circular reference like this.